site/sitecontent/terms.md

```diff
@@ -0,0 +1,183 @@
+
+### Project Licenses
+
+Only projects which are free and open are allowed on LibreCores, i.e. projects which are licensed under terms compatible with [Open Source Definition](https://opensource.org/osd-annotated) or the [Free Software Definition](https://www.gnu.org/philosophy/free-sw.html).
+
+### Inappropriate and Illegal Use
+
+LibreCores is intended for sharing open source IP, EDA tools and related projects. It is not intended to be used as an all-purpose content hosting site. LibreCores may not be used for illegal purposes, for fraudulent purposes, for the purposes of phishing (obtaining account and password information) or for Spam. Projects or Users publishing inappropriate or illegal content, using deceptive means to generate revenue or traffic, or whose primary purpose is affiliate marketing, will be suspended.
+
+### Identity Theft and Privacy
+
+Users that misleadingly appropriate the identity of another person are not permitted. Users may not post other people's personally identifying or confidential information, including but not limited to credit card numbers, Social Security Numbers, and driver's and other licence numbers. You may not post information such as other people's passwords, usernames, phone numbers, addresses and e-mail addresses unless already publicly accessible on the Web.
+
+### Hate Content, Defamation, and Libel
+
+Hate speech and other objectionable content that is unlawful, defamatory, and fraudulent. Note that an allegation of defamatory expression, in and of itself, does not establish defamation. The truth or falsehood of a bit of expression is a key element in establishing defamation, and we are not in a position to make that sort of fact-based judgement. That said, if we have reason to believe that a particular statement is defamatory (a court order, for example), we will remove that statement.
+
+### Sexually Explicit Content
+
+Accounts that use to host sexually explicit or pornographic material will be suspended.
+
+### Disruptions and Exploits
+
+We will terminate accounts and block addresses of anyone who attempts unauthorized use of LibreCores.
+
+### Mass Registration and Automation
+
+Accounts registered automatically or systematically will be permanently suspended.
+Suspended Accounts
+
+If you are found to be in violation of any of the above policies, you will receive a notice via email. Unless you explain or correct your behaviour within 72 hours, your account will be suspended. We will do our best to work with you and ensure a fair outcome in all cases. We reserve the right to immediately suspend, without notice, any content, account, or IP address which we determine to be systematically generated, spam, or potentially damaging to our service or infrastructure. Public access to the content of a suspended account will be blocked, and the LibreCores pages replaced with a generic page, or redirected to the LibreCores front page. The ability to add/edit content will also be disabled.
```

Comment:
Do we really need to be that specific here? e.g. "redirected to the LibreCores front page" sounds overly specific.